cpp/include/cuml/experimental/fil/decision_forest.hpp

```diff
@@ -227,6 +231,7 @@ struct decision_forest {
    * larger values.
    */
   void predict(
+    predict_t predict_type,
```

Comment:
Could we make this an optional parameter with a sensible default wherever possible?

Reply:
Done